engines/base/pom.xml

```diff
@@ -56,6 +56,11 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
+
```

Comment:
Why the switch from collections to guava?

Reply:
guava contains collections already, 1.0.0 was very old version, caused problems in other projects (ai-renditions)